components/datetime/src/provider/date_time.rs

```diff
@@ -232,6 +232,6 @@ pub(crate) trait ZoneSymbols<'data> {
 
 impl<'data> ZoneSymbols<'data> for () {
     fn get_payloads(&self) -> TimeZoneDataPayloadsBorrowed<'data> {
-        TimeZoneDataPayloadsBorrowed::empty()
+        TimeZoneDataPayloadsBorrowed::default()
```

Comment:
Nit: I prefer `::empty()` since "default" could mean either with-compiled-data or empty.

Reply:
If there's a default call in a method that's not marked `compiled_data`, it cannot be compiled data. This is derivable so it simplifies the code.